src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoCommon.cs

```diff
@@ -15,21 +16,21 @@ namespace System.Text.Json
     /// </summary>
     internal abstract class JsonPropertyInfoCommon<TClass, TDeclaredProperty, TRuntimeProperty, TConverter> : JsonPropertyInfo
     {
-        public Func<object, TDeclaredProperty> Get { get; private set; }
-        public Action<object, TDeclaredProperty> Set { get; private set; }
+        public Func<object?, TDeclaredProperty>? Get { get; private set; }
+        public Action<object?, TDeclaredProperty>? Set { get; private set; }
 
-        public Action<TDeclaredProperty> AddItemToEnumerable { get; private set; }
+        public Action<TDeclaredProperty>? AddItemToEnumerable { get; private set; }
```

Comment:
**Note to self:** It doesn't look like anyone calls/uses this. Remove?

Reply:
removed